src/Games/NexusMods.Games.BethesdaGameStudios/PluginOrderFile.cs

```diff
@@ -16,7 +16,7 @@
 namespace NexusMods.Games.BethesdaGameStudios;
 
 [JsonName("BethesdaGameStudios.PluginFile")]
```

Comment:
I think we can rename the Json name also, since we're still at the development phase

Reply:
Updated it, as long as the others don't disapprove